packages/newspringchurchapp/src/tabs/connect/Connect.js

```diff
@@ -15,6 +15,7 @@ import {
   Icon,
   PaddedView,
 } from '@apollosproject/ui-kit';
+import MyPrayers from '../../ui/MyPrayer';
```

Comment:
I'm not 100% excited about the naming here.

Reply:
Using `my` is what I don't think is necessary.

Reply:
I'm going to change it to `UserPrayers`. I think this is more descriptive of the intent.